bigtable/docs/snippets.py

```diff
@@ -66,6 +67,7 @@ class Config(object):
 
 
 def setup_module():
+    global INSTANCES_TO_DELETE
```

Comment:
`global` is unnecessary here, as you don't need (or want) to rebind the name, but rather to reuse the existing list.